kythe/cxx/indexer/cxx/KytheGraphObserver.h

```diff
@@ -384,7 +386,8 @@ class KytheGraphObserver : public GraphObserver {
   /// \brief Configures the starting context.
   /// \param context Context to use when the main source file is entered.
   void set_starting_context(const PreprocessorContext& context) {
-    starting_context_ = context;
+    starting_context_ =
+        CompressString(absl::StrCat(build_config_, ":", context));
```

Comment:
As we'd discussed offline, I think this will cause the indexer to get upset because it won't be able to match its context against the context recorded by the extractor.

Reply:
Fair enough :-)